tests/lock_test.py

```diff
@@ -145,10 +145,10 @@ def test_command(command, directories):
         to acquire the lock on in sorted order
 
     """
-    locks = [lock.lock_dir(directory) for directory in directories]
+    locks = [filelock.lock_for_dir(directory) for directory in directories]
```

Comment:
The function `filelock.lock_for_dir` corresponds to the old `lock.lock_dir`, which purpose is to generate a lockfile instance for the given dir path. So in theory it is the same logic.

In practice, I add hard time to find why this test was failing because I subtly changed the message in the error when lock cannot be acquired. So I am pretty sure that the test is still correctly executed.